src/pages/components/table.mdx

```diff
@@ -193,6 +193,51 @@ import TableVirtualizedCode from '!!raw-loader!../../examples/table/TableVirtual
 
 ## API
 
+The Table component follows this structure:
+
+```jsx
+<Table>
+  <Caption />
+  <Head>
+    <HeaderRow>
+      <HeaderCell />
+      <HeaderCell />
+      <SortableCell /> {/* Optional */}
+      {/* etc. */}
+    </HeaderRow>
+  </Head>
+  <Body>
+    <GroupRow>
+      {' '}
```

Comment:
Is there supposed to be a space here? What does it do?

Reply:
Oops. Not sure where this came from. I'll remove it. Thanks for catching!